circuits/test/gfmulx/polyval_mulx.test.ts

```diff
@@ -0,0 +1,227 @@
+import chai, { assert, expect } from "chai";
+import { WitnessTester } from "circomkit";
+import { bitArrayToHex, circomkit, hexToBitArray } from "../common";
+
+// Disable truncation of arrays in error messages
+chai.config.truncateThreshold = 0;
```

Comment:
Better to have this in a single test-setup file than at the start of each file imo.
We can create a `test/setup.ts` file, then either include it with `--require ./test/setup.js` or in `mocha.opts` (Which is better option)